DEVELOPERS.md

```diff
@@ -0,0 +1,125 @@
+CASDA VO Tools - Developer Documentation
+==============
+
+
+This web application provides access to CASDA data using Virtual Observatory protocols. It is designed to be reusable 
+both within CSIRO and by other data centers. As a result there should be no 
+CSIRO specific code or functionality included in the application. Any customisation should also be accessible using 
+configuration rather than via code.  
+
+Setting up
+----------
+
+This project assumes that Eclipse / STS is being used. Having checked out the project from git into the standard project location (ie: 'C:\Projects\Casda'), you can import it into Eclipse as a Gradle project. You will then need to right-click on the project and do a Gradle -> Refresh All. 
```

Comment:
do we still need to mention STS, I've stopped using that almost immediately and had no problems